programs/locker/src/instructions/v3/close_vesting_escrow.rs

```diff
@@ -0,0 +1,137 @@
+use crate::*;
+use anchor_spl::{
+    memo::Memo,
+    token_interface::{close_account, CloseAccount, Mint, TokenAccount, TokenInterface},
+};
+use util::{
+    close, harvest_fees, is_closed, parse_remaining_accounts, transfer_to_user_v3, AccountsType,
+    MemoTransferContext, ParsedRemainingAccounts, TRANSFER_MEMO_CLOSE_ESCROW,
+};
+
+/// Accounts for [locker::close_vesting_escrow].
+#[derive(Accounts)]
+#[event_cpi]
+pub struct CloseVestingEscrowV3<'info> {
+    /// Escrow.
+    #[account(
+        mut,
+        has_one = creator,
+        close = creator,
+    )]
+    pub escrow: AccountLoader<'info, VestingEscrowV3>,
+
+    /// CHECK: escrow metadata
+    #[account(
+        mut,
+        seeds = [
+            b"escrow_metadata".as_ref(),
+            escrow.key().as_ref()
+        ],
+        bump,
+    )]
+    pub escrow_metadata: AccountInfo<'info>,
+
+    /// Mint.
+    #[account(mut)]
+    pub token_mint: Box<InterfaceAccount<'info, Mint>>,
+
+    /// CHECK: Escrow Token Account.
+    #[account(mut)]
+    pub escrow_token: AccountInfo<'info>,
+
+    #[account(mut)]
+    pub creator_token: Box<InterfaceAccount<'info, TokenAccount>>,
+
+    /// Creator.
+    #[account(mut)]
+    pub creator: Signer<'info>,
+
+    /// Token program.
+    pub token_program: Interface<'info, TokenInterface>,
+
+    /// Memo program.
+    pub memo_program: Program<'info, Memo>,
+}
+
+pub fn handle_close_vesting_escrow_v3<'c: 'info, 'info>(
```

Comment:
Do we support cancel_mode in escrow_v3? should we?

Reply:
I think w should close. 
In case, all recipients claimed all locked amount. Then, we should close escrow_v3 to harvest fee.